### PR TITLE
Feature user crud

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding user CRUD operations
## Trello Card
https://trello.com/c/CbXkALre
## Test
image

### CODE REVIEW

File: src/main/java/com/wolox/training/controllers/UserController.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    @Autowired
+    private UserService userService;
+
+    @PostMapping
+    public @ResponseBody
+    ResponseEntity<Users> create(@RequestBody UsersVO usersVO) {
+        return new ResponseEntity<>(userService.create(usersVO), HttpStatus.CREATED);
+    }
+
+    @PostMapping("/{id}/addBook")
```
Comment (torsello) [inline src/main/java/com/wolox/training/controllers/UserController.java]: I think that in this case, and in the removeBook, the method could be an Patch or a Put maybe.

We can reach an agreement, but for Post method, you have to use a RequestBody

**Review by torsello — CHANGES_REQUESTED** [review]

File: src/main/java/com/wolox/training/controllers/UserController.java (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/controllers/UserController.java]: Thanks Mati! It slipped my mind! :)

**Review by torsello — APPROVED** [review]
Looks Good To Me :100:

File: src/main/java/com/wolox/training/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    @PostMapping
+    public @ResponseBody
+    ResponseEntity<Users> create(@RequestBody UsersVO usersVO) {
+        return new ResponseEntity<>(userService.create(usersVO), HttpStatus.CREATED);
+    }
+
+    @PatchMapping("/{id}/addBook")
+    public ResponseEntity<Users> addBook(@PathVariable(name = "id") Long userId, @RequestParam Long bookId) {
+        Users user = userService.addUserBook(userId, bookId);
```
Comment (jdhiguita) [inline src/main/java/com/wolox/training/controllers/UserController.java]: A linter plugin say: It's no necessary to declare variable to be return.

**Review by jdhiguita — COMMENTED** [review]
Great job! One single comment

File: src/main/java/com/wolox/training/controllers/UserController.java (hunk 2 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/controllers/UserController.java]: Cool! I pushed the change

**Review by jdhiguita — APPROVED** [review]

**Pull request merged**